src/components/views/SuggestedViews/ClickableCard.tsx

```diff
@@ -1,39 +1,21 @@
 import { ReactEventHandler } from 'react';
-import { Card, CardActionArea, makeStyles } from '@material-ui/core';
+import { Box, Card, CardActionArea } from '@material-ui/core';
 
 interface ZetkinCardProps {
   onClick?: ReactEventHandler;
   value?: string | number;
 }
 
-const useStyles = makeStyles((theme) => ({
-  card: ({ hasClickHandler }: { hasClickHandler: boolean }) => ({
-    padding: theme.spacing(hasClickHandler ? 0 : 2),
-  }),
-  cardActionArea: {
-    padding: theme.spacing(2),
-  },
-}));
-
```

Comment:
These styles can be safely removed because the only place a card without a click handler was being used the `JourneyInstanceCard`. We actually want that component to have a card action area now, because the entire component is a clickable button.

Reply:
Thanks for explaining that!